ResearchKit/Common/ORKHelpers.m

```diff
@@ -247,8 +247,26 @@ void ORKEnableAutoLayoutForViews(NSArray *views) {
 }
 
 NSBundle *ORKBundle() {
-    NSBundle *bundle = [NSBundle bundleForClass:[ORKStep class]];
-    return bundle;
+    static NSBundle *__bundle;
```

Comment:
Shouldn't we avoid the *double underscore prefix*? It's normally [reserved](http://stackoverflow.com/a/10687114/269753) in *C* and *C++*.

It's used on several variable names on `ORKHelpers.m`, we could review the file and remove them.